datalab/settings.py

```diff
@@ -134,7 +134,8 @@ def get_list_from_env(variable, default=None):
 DATALAB_OPERATION_BUCKET = os.getenv('DATALAB_OPERATION_BUCKET', 'datalab-operation-output-bucket')
 
 # Datalab Archive
-ARCHIVE_API = os.getenv('ARCHIVE_API', 'https://datalab-archive.photonranch.org')
+ARCHIVE_API = os.getenv('ARCHIVE_API', 'https://archive-api.lco.global')
+AUTH_TOKEN = os.getenv('AUTH_TOKEN')
```

Comment:
can we call this ARCHIVE_AUTH_TOKEN (or I like ARCHIVE_API_TOKEN)? We may have more services/tokens in the future.

Reply:
so for the memmapping astropy looks like it does it by default : [handling large files](https://docs.astropy.org/en/stable/io/fits/#working-with-large-files) and [default args](https://docs.astropy.org/en/stable/io/fits/api/files.html#open) so whatever I was doing was redundant.